python_modules/libraries/dagster-fivetran/dagster_fivetran_tests/conftest.py

```diff
@@ -542,6 +564,51 @@ def fetch_workspace_data_api_mocks_fixture(
         yield response
 
 
+@pytest.fixture(
+    name="fetch_workspace_data_multiple_connectors_mocks",
+)
+def fetch_workspace_data_multiple_connectors_mocks_fixture(
+    connector_id: str, destination_id: str, group_id: str
+) -> Iterator[responses.RequestsMock]:
+    with responses.RequestsMock() as response:
+        response.add(
+            method=responses.GET,
+            url=f"{FIVETRAN_API_BASE}/{FIVETRAN_API_VERSION}/groups",
+            json=SAMPLE_GROUPS,
+            status=200,
+        )
+
+        response.add(
+            method=responses.GET,
+            url=f"{FIVETRAN_API_BASE}/{FIVETRAN_API_VERSION}/groups/{group_id}/connectors",
+            json=SAMPLE_CONNECTORS_FOR_GROUP_WITH_EXTRA_CONNECTOR,
+            status=200,
+        )
+
+        response.add(
+            method=responses.GET,
+            url=f"{FIVETRAN_API_BASE}/{FIVETRAN_API_VERSION}/destinations/{destination_id}",
+            json=SAMPLE_DESTINATION_DETAILS,
+            status=200,
+        )
+
+        response.add(
+            method=responses.GET,
+            url=f"{get_fivetran_connector_api_url(connector_id)}/schemas",
+            json=SAMPLE_SCHEMA_CONFIG_FOR_CONNECTOR,
+            status=200,
+        )
+
+        response.add(
+            method=responses.GET,
+            url=f"{get_fivetran_connector_api_url(EXTRA_TEST_CONNECTOR_ID)}/schemas",
+            json=SAMPLE_SCHEMA_CONFIG_FOR_EXTRA_CONNECTOR,
+            status=200,
+        )
+
+        yield response
```

Comment:
I believe we can reuse the `fetch_workspace_data_api_mocks` fixture here

```suggestion        
@pytest.fixture(
    name="fetch_workspace_data_multiple_connectors_mocks",
)
def  fetch_workspace_data_multiple_connectors_mocks_fixture(
    fetch_workspace_data_api_mocks: responses.RequestsMock,
) -> Iterator[responses.RequestsMock]:
    fetch_workspace_data_api_mocks.add(
        method=responses.GET,
        url=f"{get_fivetran_connector_api_url(EXTRA_TEST_CONNECTOR_ID)}/schemas",
        json=SAMPLE_SCHEMA_CONFIG_FOR_EXTRA_CONNECTOR,
        status=200,
    )
    yield fetch_workspace_data_api_mocks
```